3.Tests/UI/StepDefinitions/Settings/Profile/UpdateFullNameStepDefinition.cs

```diff
@@ -0,0 +1,69 @@
+﻿using OpenQA.Selenium;
+using SeleniumTest.Core.Drivers;
+using SeleniumTest.Core.Interfaces;
+using SeleniumTest.Tests.Steps.Commons;
+using TechTalk.SpecFlow;
+using Views.WebAppPages;
+
+namespace SeleniumTest.Tests;
+
+[Binding]
+[Scope(Feature = "Full Name update")]
+[TestFixture]
+public class UpdateFullNameStepDefinition : CommonSteps
+{
+    private readonly ProfilePage _profile;
+    private readonly IGenericWebDriver _driver;
+    private readonly ScenarioContext _scenarioContext;
+
+    public UpdateFullNameStepDefinition(ScenarioContext scenarioContext, ChromeWebDriver driver) : base(scenarioContext, driver)
+    {
+        _scenarioContext = scenarioContext;
+        _driver = driver;
+        _profile = new ProfilePage(driver);
+    }
+
+    [Given(@"the user clicks on the Settings option on the Nav Bar")]
+    public void GiventheuserclicksontheSettingsoptionontheNavBar()
+    {
+        _profile.Settings.Click();
```

Comment:
Settings button should be on another NavBar component, so no need to instantiate profile page yet

Reply:
Fixed. As I mentioned in the first comment I added that element in the Home Page.